### PR TITLE
fix flickering in team's details

### PR DESCRIPTION
**Pull request by olexii4**
Signed-off-by: Oleksii Orel <oorel@codenvy.com>

### What does this PR do?
Fix flickering in team's details.

https://github.com/codenvy/codenvy/issues/1756

#### Changelog
Fixed flickering when changing tabs in team's details

#### Release Notes
Fixed flickering when changing tabs in team's details

#### Docs PR
N/A

### CODE REVIEW

File: dashboard/src/app/teams/team-details/team-details.service.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+ * patents in process, and are protected by trade secret or copyright law.
+ * Dissemination of this information or reproduction of this material
+ * is strictly forbidden unless prior written permission is obtained
+ * from Codenvy S.A..
+ */
+'use strict';
+import {CodenvyTeam} from '../../../components/api/codenvy-team.factory';
+import {CodenvyUser} from '../../../components/api/codenvy-user.factory';
+
+export class TeamDetailsService {
```
Comment (benoitf) [inline dashboard/src/app/teams/team-details/team-details.service.ts]: maybe explain what the service do ?

Comment (olexii4) [inline dashboard/src/app/teams/team-details/team-details.service.ts]: done

File: dashboard/src/app/teams/team-details/team-members/list-team-members.controller.ts [hunk 2]
```diff
@@ -128,22 +130,32 @@ export class ListTeamMembersController {
...
     this.isNoSelected = true;
 
+    this.owner = teamDetailsService.getOwner();
+    this.team  = teamDetailsService.getTeam();
+
     this.fetchMembers();
   }
 
   /**
    * Fetches the lis of team members.
```
Comment (akurinnoy) [inline dashboard/src/app/teams/team-details/team-members/list-team-members.controller.ts]: 'the lis of' - typo?

File: dashboard/src/app/teams/team-details/team-details.controller.ts [hunk 3]
```diff
@@ -169,50 +167,24 @@ export class TeamDetailsController {
...
-   * Fetches the team's details by it's name.
-   */
-  fetchTeamDetails(): void {
-    this.team  = this.codenvyTeam.getTeamByName(this.teamName);
-
-    if (!this.team) {
-      this.codenvyTeam.fetchTeamByName(this.teamName).then((team: any) => {
-        this.team = team;
-        this.newName = angular.copy(this.team.name);
+    this.team = teamDetailsService.getTeam();
```
Comment (ashumilova) [inline dashboard/src/app/teams/team-details/team-details.controller.ts]: this part causes bug with typing another name in the address line:
image

Comment (olexii4) [inline dashboard/src/app/teams/team-details/team-details.controller.ts]: Sorry. Didn't check case with wrong URL.
Done

**Pull request merged**